index.html

```diff
@@ -17,21 +17,30 @@
     </script>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="style.css" rel="stylesheet">
+    <script src="script.js" defer></script>
     <title>Sahar-Abdel Samad's Portfolio</title>
 </head>
 
 <body>
     <header class="toolbar">
```

Comment:
- [ ] [Optional] Just an optional requirement, you can decide whether you implement it or not depending on the free band width you have; the navbar is too transparent hence it kills the usability of your page, because when we scroll down, some letters are overlaying one another, would you address that, kindly watch the following screencast for more context!

https://user-images.githubusercontent.com/56886064/145975333-28a43b14-6089-402f-9fad-41b2f2857efe.mp4

Reply:
Thank you for your feedback:star_struck: I fixed it. It's way much better like this:smiley: